quic/client/handshake/FizzClientHandshake.cpp

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+#include <quic/client/handshake/FizzClientHandshake.h>
+
+#include <quic/handshake/FizzCryptoFactory.h>
+
+namespace quic {
+
+FizzClientHandshake::FizzClientHandshake(
+    QuicCryptoState& cryptoState,
+    std::shared_ptr<FizzClientContext> fizzContext)
+    : ClientHandshake(cryptoState), fizzContext_(std::move(fizzContext)) {}
+
+void FizzClientHandshake::connect(
+    std::shared_ptr<const fizz::client::FizzClientContext> context,
```

Comment:
this is the kind of fun stuff that makes it confusing, the class takes a FizzClientContext as a ctor param as well as a connect() param :) but they're actually 2 different types of objects